macros/materializations/model.sql

```diff
@@ -54,18 +54,24 @@
     {%- set ml_config = config.get('ml_config', {}) -%}
     {%- set raw_labels = config.get('labels', {}) -%}
     {%- set sql_header = config.get('sql_header', none) -%}
-
+    
```

Comment:
Whitespace

Reply:
Should be fixed now as well as issue with inconsistent use of quotes